schemas/expconf/v0/resources.json

```diff
@@ -52,13 +52,6 @@
             "maximum": 99,
             "default": null
         },
-        "resource_manager": {
```

Comment:
I'd have expected that this results in an invalid experiment config that had a section
```yaml
resources:
  resource_manager: 'agent'
```
but when I try running an experiment with a section like this it seems to launch just fine. Is this patch backwards compatible in a way I don't understand?

Reply:
huh I'll double-check -- I ran 'git revert' on the three commits I wanted to revert, so this is unexpected

Reply:
this change wasn't actually publicly released. does this cause any issues..

are there places with serialize then deserialize an experiment config that are going to pick this up and then explode?

Reply:
@carolinaecalderon to test this you may want to run an experiment before this revert, pause it, shutdown your cluster, add this revert, recompile, restart the cluster and see if resuming the experiment works.

if that works i think this is safe even though it isn't actually backward compatible.

Reply:
@wes-turner it's possible that when you tested this I hadn't re-generated the schemas, because when I run a test following @stoksc 's proposed flow, I get this error:
`ERRO[2024-03-06T15:15:46-05:00] failed to restore experiment: 1               error="cannot restore experiment 1 with unparsable config"`
tldr; the change is not backwards compatible since the experiment can't resume. However, since this change was not announced to the customer and is being reverted one release later, would it be fine to remove it?

Reply:
Wait no, that is exactly the error I was concerned about. You launched an experiment that did not use the multirm features and upgraded and it broke.

Reply:
> this change wasn't actually publicly released. does this cause any issues..

Nice, nice, nice. No issues, no deserialization that I know about.

Thanks. The public deprecation was exactly my concern.

Reply:
Turns out that doing this _does_ break experiment restore, so @carolinaecalderon is going to put up a fix. Really good you asked this @wes-turner, thanks.